src/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+import "./CourseCard.css";
+
+const CourseCard = ({ product }) => {
+  const navigate = useNavigate();
+
+  return (
+    <div onClick={() => navigate(`/courses`)} className="course-card">
```

Comment:
Suggestion: You could abstract the function out of JSX syntax, it would make code more readable, and managing that function would be easier.